source/_integrations/spokestack_tts.markdown

```diff
@@ -0,0 +1,74 @@
+---
+title: "Spokestack Text-to-Speech (TTS)"
+description: "Spokestack TTS platform documentation"
+ha_release: "0.38"
+ha_category: Text-to-speech
+ha_iot_class: "Cloud Push"
+ha_codeowners:
+  - '@spokestack'
+  - '@will-rice'
+  - '@space-pope'
+ha_domain: spokestack_tts
+---
+
+[Spokestack](https://www.spokestack.io/) offers open-source libraries for adding a voice interface to _anything_. `spokestack_tts` is a text-to-speech platform that uses our Python [library](https://github.com/spokestack/spokestack-python) to read text with natural sounding custom voices. Usage of TTS with the default voice is free, but the API requires authentication. If you already have an account, [log in](https://www.spokestack.io/login), otherwise you will need to [create](https://www.spokestack.io/create) one. The credentials can be found in your [account settings](https://www.spokestack.io/account/settings).
```

Comment:
The fact it uses a specific Python library isn't interesting for the end-user.

```suggestion
[Spokestack](https://www.spokestack.io/) offers open-source libraries for adding a voice interface to _anything_. `spokestack_tts` is a text-to-speech platform to read text with natural-sounding custom voices. Usage of TTS with the default voice is free, but the API requires authentication.

If you already have an account, [log in](https://www.spokestack.io/login), otherwise you will need to [create](https://www.spokestack.io/create) one. The credentials can be found in your [account settings](https://www.spokestack.io/account/settings).
```